internal/installcfg/installcfg.go

```diff
@@ -36,8 +36,22 @@ type baremetal struct {
 }
 
 type platform struct {
-	Baremetal *baremetal    `yaml:"baremetal,omitempty"`
-	None      *platformNone `yaml:"none,omitempty"`
+	Baremetal *baremetal       `yaml:"baremetal,omitempty"`
+	None      *platformNone    `yaml:"none,omitempty"`
+	Vsphere   *platformVsphere `yaml:"vsphere,omitempty"`
+}
+
+type platformVsphere struct {
```

Comment:
Are all those fields are mandatory? maybe we can add remarks for the mandatory fields

Reply:
@eliorerz can you validate it?

Reply:
Just the `Folder` is not mandatory